client/marketing/overview/welcome-card/style.scss

```diff
@@ -46,14 +52,20 @@
 	}
 
 	h3 {
-		font-size: 24px;
-		line-height: 32px;
+		font-size: 20px;
+		line-height: 26px;
 		font-weight: normal;
-		margin-left: 20px;
+		text-align: center;
+		margin: 1em 0 0;
```

Comment:
Thanks @danielbitzer this is looking much better! One thing I noticed (Not sure if it was intended) but the margin here causes the text to become out of center alignment (pushed down) for large screens - it ends up looking like 

![Screen Shot 2020-04-15 at 3 21 11 pm](https://user-images.githubusercontent.com/355014/79303112-42d91400-7f2d-11ea-89bb-7ac78e14e7e7.png)

vs

![Screen Shot 2020-04-15 at 3 20 56 pm](https://user-images.githubusercontent.com/355014/79303121-49678b80-7f2d-11ea-9755-6745dbfa2c9d.png)

Reply:
Good spot, yes that was a mistake.

Reply:
I've just fixed that issue and another alignment issue with the installedextensions card